contracts/program-registry/src/state.rs

```diff
@@ -0,0 +1,6 @@
+use cosmwasm_std::Binary;
+use cw_storage_plus::{Item, Map};
+
+pub const LAST_ID: Item<u64> = Item::new("id");
+pub const WORKFLOWS: Map<u64, Binary> = Map::new("programs");
+pub const WORKFLOWS_BACKUP: Map<u64, Binary> = Map::new("programs_backups");
```

Comment:
```suggestion
pub const PROGRAMS: Map<u64, Binary> = Map::new("programs");
pub const PROGRAM_BACKUP: Map<u64, Binary> = Map::new("program_backups");
```

guessing these are leftovers?

Reply:
Fixed, missed these